.github/workflows/dry-run.yml

```
@@ -13,14 +13,13 @@
-        uses: "actions/setup-node@v3"
+      - name: Setup Bun
+        uses: oven-sh/setup-bun@v1
         with:
-          node-version: 18.15
-          cache: 'yarn'
+          bun-version: latest
 
       - name: Install dependencies and build
-        run: yarn install && yarn build && yarn run bootstrap
+        run: bun install && bun run build && bun run bootstrap
         
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)